### PR TITLE
Issue #1682 Fix rendering of asciidoctor diagram in docs.openshift.org

### LINKED ISSUE #1682: Minishift architecture diagram is not rendering due to permission issue
Seem to be permission issue 
screenshot from 2017-11-10 16-02-53

### PR DESCRIPTION
**Pull request by budhrg**
Fix #1682

### CODE REVIEW

File: docs/source/using/basic-usage.adoc [hunk 1]
```diff
@@ -31,6 +31,8 @@ xref:../openshift/openshift-client-binary.adoc#openshift-client-binary-overview[
 For more details about using {project} to manage your local OpenShift cluster, see the xref:../openshift/index.adoc#[Interacting with OpenShift] section.
 
 [[diagram-minishift-architecture]]
+
+//DOC_TAR_MARKER_TITLE_Minishift architecture_IMG_NAME_minishift-architecture_ENDS
```
Comment (budhrg) [inline docs/source/using/basic-usage.adoc]: @hferentschik , Please let me know if you have any other idea to tackle this issue. I could come up with this idea.

File: .gitignore [hunk 2]
```diff
@@ -44,7 +44,9 @@ docs/build
 docs/source/_tmp/*.md
 docs/source/command-ref/*.adoc
 docs/source/**/variables.adoc
-docs/source/using/images/minishift-architecture*
+docs/source/using/images/*
```
Comment (hferentschik) [inline .gitignore]: -1, what if we are going to add some static images

**Comment by budhrg** [comment]
retest this please

File: .gitignore [hunk 3]
```diff
@@ -44,7 +44,9 @@ docs/build
...
 docs/.bundle/
 docs/.sass-cache/
 docs/.java
+docs/source/using/.asciidoctor
+docs/?/
```
Comment (hferentschik) [inline .gitignore]: This is something we need to fix, not just ignore. We need to find out what creates this file.

File: docs/Rakefile [hunk 4]
```diff
@@ -86,7 +86,13 @@ task :adoc_tar => [:init, :adoc_variables, :markdown_to_asciidoc] do
         tar.add_file(f, 0444) { |io|
           if File.extname(f) == ".adoc" then
             adoc_content = File.read(f)
-            adoc_content = adoc_content.gsub(/include::.*variables\.adoc\[\]/, "include::minishift/variables.adoc[]")
+            adoc_content.gsub!(/include::.*variables\.adoc\[\]/, "include::minishift/variables.adoc[]")
+            # skip writing-docs.adoc file since it contains match regex pattern
```
Comment (hferentschik) [inline docs/Rakefile]: what is this about? No better way to fix that. Having to exclude writing-docs explicitly sucks.

File: docs/source/contributing/writing-docs.adoc [hunk 5]
```diff
@@ -45,6 +45,23 @@ See the link:https://github.com/openshift/openshift-docs/blob/master/contributin
...
 
+[[adding-diagram]]
+== Adding Diagram
+
+link:http://asciidoctor.org/docs/asciidoctor-diagram[Asciidoctor Diagram] is a set of Asciidoctor extensions that enable you to add diagrams, which you describe using plain text, to your AsciiDoc document. You can refer link:http://asciidoctor.org/docs/asciidoctor-diagram/#creating-a-diagram[here] in more details on how to create a diagram.
+
+To integrate this diagram properly with xref:integration-with-docs-openshift-org[docs.openshift.org], you need to wrap the _anatomy of your diagram_ as following:
+
+```
+//DOC_TAR_MARKER_TITLE_<title>_IMG_NAME_<image-name>_ENDS
```
Comment (hferentschik) [inline docs/source/contributing/writing-docs.adoc]: ????

File: docs/source/using/basic-usage.adoc (hunk 1 above)
Comment (hferentschik) [reply docs/source/using/basic-usage.adoc]: I don't know what you are trying to do here to be honest

File: docs/source/contributing/writing-docs.adoc (hunk 5 above)
Comment (budhrg) [reply docs/source/contributing/writing-docs.adoc]: @hferentschik https://github.com/minishift/minishift/issues/1682#issuecomment-344352588 Root cause of image rendering in docs.openshift.org

Comment (hferentschik) [reply docs/source/contributing/writing-docs.adoc]: -1

**Comment by budhrg** [comment]
Closing this PR as we are not going down this path.

**Pull request closed without merging**